### ISSUE TITLE
Stopping and starting has issues

### ISSUE THREAD

**Issue by LiamDawe** [josh-richardson/cadmus#4]
When stopping it and then starting it again, it doesn't seem to be able to bring it back up.

Noticed this in terminal:
```Traceback (most recent call last):
  File "main.py", line 102, in <module>
  File "main.py", line 22, in <lambda>
  File "main.py", line 62, in enable_noise_suppression
  File "main.py", line 43, in load_modules
  File "pulsectl/pulsectl.py", line 655, in module_load
pulsectl.pulsectl.PulseOperationInvalid: Bad state [pulse errno 15]
/home/liam/.cache/cadmus
/home/liam/.cache/cadmus/librnnoise_ladspa.so
Traceback (most recent call last):
  File "pulsectl/pulsectl.py", line 654, in module_load
  File "pulsectl/_pulsectl.py", line 665, in _wrapper
pulsectl._pulsectl.LibPulse.CallError: ('pa_context_load_module', (<pulsectl._pulsectl.LP_PA_CONTEXT object at 0x7f7c05b0e0d0>, b'module-null-sink', b'sink_name=mic_denoised_out', <CFunctionType object at 0x7f7bf5378048>, None), <pulsectl._pulsectl.LP_PA_OPERATION object at 0x7f7c05b0ee18>, 'Bad state [pulse errno 15]')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 102, in <module>
  File "main.py", line 22, in <lambda>
  File "main.py", line 62, in enable_noise_suppression
  File "main.py", line 43, in load_modules
  File "pulsectl/pulsectl.py", line 655, in module_load
pulsectl.pulsectl.PulseOperationInvalid: Bad state [pulse errno 15]```

**Comment by josh-richardson** [comment]
This is an interesting one. I'm not sure why it happens, but I think it could be due to the fact that I'm making synchronous calls to PulseAudio. If you exit the application, and run the following in a Terminal:

```
pactl unload-module module-loopback
pactl unload-module module-null-sink
pactl unload-module module-ladspa-sink
```

It should purge everything that it's done. If you then re-start it, it ought to be relatively happy. This is definitely something I'll be looking into though.

**Comment by LiamDawe** [comment]
Yeah, I can actually just entirely close the app and then load it again and it works. So it's something in the app itself that can't seem to re-do it.

Impressive though BTW, just ran a test and it really does bring down sounds even right next to the microphone. Wonderful to have such a one-click solution.

**Comment by josh-richardson** [comment]
This has been addressed in #26 and as such I'm closing this issue. Please do re-open it if the new version doesn't fully solve this issue.
